programs/fuzz/fuzz_client.c

```diff
@@ -175,6 +182,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
     mbedtls_ctr_drbg_free(&ctr_drbg);
     mbedtls_ssl_config_free(&conf);
     mbedtls_ssl_free(&ssl);
+    mbedtls_psa_crypto_free();
```

Comment:
Sorry for not noticing earlier, but shouldn't this be guarded by `MBEDTLS_USE_PSA_CRYPTO` as well? (For all programs.)

Reply:
Yes. (Technically it'll work as long as `MBEDTLS_PSA_CRYPTO_C` is enabled, but you'll get a link error if that's disabled, which I'd expect the CI to catch.)